core/src/main/scala/org/broadinstitute/dsde/rawls/billing/BillingProfileManagerDAO.scala

```diff
@@ -17,7 +17,10 @@ import scala.jdk.CollectionConverters._
 trait BillingProfileManagerDAO {
   def createBillingProfile(displayName: String, billingInfo: Either[RawlsBillingAccountName, AzureManagedAppCoordinates], userInfo: UserInfo): Future[ProfileModel]
 
-  def listBillingProfiles(samUserResources: Seq[SamUserResource], userInfo: UserInfo)(implicit ec: ExecutionContext): Future[Seq[RawlsBillingProject]]
+  /***
+    * Uses BPM to retrieve billing profile information for each of the RawlsBillingProjects (via their billingProjectIDs).
+    */
+  def populateBillingProfiles(samUserResources: Seq[SamUserResource], userInfo: UserInfo, rawlsBillingProjects: Seq[RawlsBillingProject])(implicit ec: ExecutionContext): Future[Seq[RawlsBillingProject]]
```

Comment:
This method was originally documented as returning all the profiles that a user has access to. However, I really only need to look up the profiles for the Rawls projects the user has access to, using the stored billingProjectIDs.

I suppose the alternative is to return all profiles, and then match them up with the rawlsBillingProjects after the fact.

Reply:
@aherbst-broad what was your thought about this? Do you think we should keep this method as returning all billing profiles the user has access to?

Reply:
@aherbst-broad I went back to returning all billing profileModels the user has access to in the commit I just pushed up.